modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/fullscreen_source_editor.js

```diff
@@ -135,9 +135,11 @@ AUI.add(
 				_onEditorChange(event) {
 					var instance = this;
 
-					instance._previewPanel.html(
-						instance._getHtml(event.newVal)
-					);
+					if (event.newVal) {
+						instance._previewPanel.html(
+							instance._getHtml(event.newVal)
+						);
+					}
```

Comment:
@NemethNorbert Is there a way to prevent breakage, instead of handling it? Ideally setting of html should not be invoked when you click on the preview panel, only when you change content of the source panel.